### PR TITLE
Handle PkgEval request failures

### PR DESCRIPTION
**Pull request by vdayanand**

### CODE REVIEW

File: src/DocumentationGenerator.jl [hunk 1]
```diff
@@ -147,10 +147,13 @@ function get_pkg_eval_data()
 
         if occursin(r"\d{4}\-\d{2}/\d{2}", latest_date)
             last_db_url = "https://raw.githubusercontent.com/JuliaCI/NanosoldierReports/master/pkgeval/by_date/$(latest_date)/db.json"
-
-            resp = HTTP.get(last_db_url)
```
Comment (pfitzseb) [inline src/DocumentationGenerator.jl:151]: We should be able to just pass `status_exception = false` to the `HTTP.get` call. I always forget that's not the default...

Comment (vdayanand) [reply src/DocumentationGenerator.jl:151]: `status_exception` should work. But try catch handles DNS errors too even though they are unlikely to occur.

**Comment by vdayanand** [comment]
Got 404
```
.ERROR: LoadError: HTTP.ExceptionRequest.StatusError(404, "GET", "/JuliaCI/NanosoldierReports/master/pkgeval/by_date/2020-06/14/db.json", HTTP.Messages.Response:
"""
HTTP/1.1 404 Not Found
Connection: keep-alive
Content-Length: 14
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
Content-Type: text/plain; charset=utf-8
Via: 1.1 varnish (Varnish/6.0)
X-GitHub-Request-Id: A25A:6F59:10AB9:22B59:5EE72731
Accept-Ranges: bytes
Date: Mon, 15 Jun 2020 07:45:53 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18174-EWR
X-Cache: MISS, MISS
X-Cache-Hits: 0, 0
X-Timer: S1592207153.253739,VS0,VE69
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
X-Fastly-Request-ID: a2a4a9351b295d032dab03d42acac1d4ad370ae9
Expires: Mon, 15 Jun 2020 07:50:53 GMT
Source-Age: 0

404: Not Found""")
Stacktrace:
```

File: src/DocumentationGenerator.jl (hunk 1 above)
Comment (pfitzseb) [reply src/DocumentationGenerator.jl:151]: Fair enough.

**Pull request merged**